src/targets/browser/index.ejs

```diff
@@ -31,6 +31,7 @@
   data-cozy-icon-path="{{.IconPath}}"
   data-cozy-subdomain-type="{{.SubDomain}}"
   data-cozy-default-wallpaper="{{.DefaultWallpaper}}"
+  data-cozy-flags="{{.Flags}}"
```

Comment:
Is it the right moment to remove all those data-cozy-* and use   `<div role="application" data-cozy="{{.CozyData}}"></div>` ? As it's the recommended way

Reply:
I plan to do it in a separate PR, as it may involve more changes. Specifically, the `{{.DefaultWallpaper}} ` is not injected in `{{.Cozy}}` which is a problem for the home.

I plan to check if it should be included or not, and if not how the home can handle this.